src/status_im/chat/views/message_input.cljs

```diff
@@ -96,16 +101,20 @@
              [plain-message/commands-button height #(set-layout-size 0)]
              [view (st/message-input-container height)
               (if @command?
-                [command-input input-options @command]
+                [command-input input-options @command sending-allowed?]
                 [message-input input-options set-layout-size])]
              [plain-message/smile-button height]
              (when (or (and @command? (not (str/blank? @input-command)))
                        @valid-plain-message?)
                (let [on-press (if @command?
                                 #(dispatch [:send-command!])
                                 plain-message/send)]
-                 [send-button {:on-press #(do (dispatch [:set-chat-ui-props :show-emoji? false])
-                                              (on-press %))}]))
+                 [send-button {:on-press (fn []
+                                           (when @sending-allowed?
+                                             (reset! sending-allowed? false)
+                                             (js/setTimeout #(reset! sending-allowed? true) 500)
+                                             (dispatch [:set-chat-ui-props :show-emoji? false])
+                                             (on-press %)))}]))
```

Comment:
WARNING: Use of undeclared Var status-im.chat.views.message-input/% at line 117 src/status_im/chat/views/message_input.cljs